src/parse_input.c

```diff
@@ -1,6 +1,6 @@
 #include "../inc/minishell.h"
 
-static int	count_slash_semicolon(char *line)
+static int	count_char_until(char *line, char c, char until)
```

Comment:
#note for me. count_the_char

Reply:
This can be deleted, I decided to not use it afterwards

Reply:
I noticed it